sssd_test_framework/roles/keycloak.py

```diff
@@ -126,12 +129,66 @@ def add(
 
         return self
 
-    def delete(self) -> None:
+    def delete(
+        self,
+    ) -> SSHProcessResult:
         """
         Delete Keycloak user.
+
+        :return: SSH exec result
+        :rtype: SSHProcessResult
         """
         del_user = f"delete users/{self.id}"
-        self.role.kcadm(del_user)
+        result = self.role.kcadm(del_user)
+        return result
+
+    def modify(
+        self,
+        *,
+        firstName: str | None = None,
+        lastName: str | None = None,
+        email: str | None = None,
+        enabled: bool | None = None,
+    ) -> KeycloakUser:
+        """
+        Modify keycloak user attributes.
+
+        Parameters that are not set are ignored.
```

Comment:
I think this is enough, like the rest of the code, that it doesn't need to be said.

Reply:
removed